### PR TITLE
add lock functionality

### LINKED ISSUE #33: Add lock functionality

### LINKED ISSUE #36: Specify and adjust role responsibilities

### PR DESCRIPTION
**Pull request by cloud8little**
close #33

### CODE REVIEW

**Review by steven1227 — APPROVED** [review]

**Comment by gsmachado** [comment]
IMHO, this PR is fine.

**Comment by cloud8little** [comment]
@mialbu  shall we merge?

File: contracts/BridgeContract.sol [hunk 1]
```diff
@@ -309,4 +328,18 @@ contract BridgeContract {
...
         return uint64(_value / (10 ** 10));
     }
+
+    // Lock the deposit process.
+    function lock() external onlyAdmin {
+        locked = true;
+    }
+
+    // UnLock the deposit process.
+    function unLock() external onlyRecoverer {
```
Comment (mialbu) [inline contracts/BridgeContract.sol]:
```suggestion
    function unlock() external onlyGovernor {
```

**Review by mialbu — CHANGES_REQUESTED** [review]
Please take a look at [this document](https://github.com/bane-labs/bridge-docs/blob/bb5884260e65aa03fb4be350a37032778b780b6f/roles.md) and change the naming of admin and recoverer in the contract.

**Comment by cloud8little** [comment]
@mialbu  all resolved in https://github.com/bane-labs/bridge-evm-contracts/pull/34/commits/0671d84dfdbf3be7529a24ab8d22d6caee6838a8

**Review by mialbu — APPROVED** [review]

**Pull request merged**